bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/UniFiControllerRequest.java

```diff
@@ -83,6 +83,10 @@
 
     private String path = "/";
 
+    private String csrfToken;
+
+    private Boolean unifiOS;
```

Comment:
See previous comments

Reply:
Same here with relation to where to place this field and to make it final.